hack/dashboard/openshift/deploy-grafana.sh

```diff
@@ -28,6 +28,24 @@ validate_cluster() {
 	header "Validating cluster"
 
 	local -i ret=0
+
+	local oc_version=""
+	oc_version=$(oc version --client -oyaml | grep releaseClientVersion | cut -f2 -d:)
+
+	local -i oc_major_version oc_minor_version
+	oc_major_version=$(echo "$oc_version" | cut -f1 -d.)
+	oc_minor_version=$(echo "$oc_version" | cut -f2 -d.)
+
+	[[ $oc_major_version -eq 4 ]] && [[ $oc_minor_version -ge 12 ]] || {
+		fail "oc version $oc_version used should be at least 4.12.0"
+		info "install a newer version of oc"
+		cat <<-EOF
+			     curl -sNL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.13.0/openshift-client-linux.tar.gz |
+				   tar -xzf - -C .
```

Comment:
can we do this `tar -xzf - -C /usr/local/bin` ?

Reply:
I wouldn't encourage putting anything in system path. It is best for users to decide where the binary should be installed.

Again, this isn't the only way to install oc.